app/views/user/_session_manager.html.erb

```diff
@@ -1,17 +1,37 @@
 <% if current_user %>
   <li class="nav-item">
-    <%= link_to "Cart ( #{current_user.line_items.size} )", cart_path, class: "nav-link" %>
+    <%= link_to cart_path, title: 'Your cart', class: "nav-link" do %>
+      <span><i class="bi bi-cart text-light"></i></span>
+      <% if current_user.line_items.size > 0 %>
+        <span class="badge badge-pill bg-danger"><%= current_user.line_items.size %></span>
+      <% end %>
+    <% end %>
   </li>
   <li class="nav-item dropdown">
-    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-      <%= current_user.email %>
+    <a class="nav-link dropdown-toggle bi bi-person-circle text-light" title="Your account" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
     </a>
     <ul class="dropdown-menu dropdown-menu-lg-end" aria-labelledby="navbarDropdown">
-      <li><%= link_to "Orders", orders_path, class:"dropdown-item" %></li>
-      <li><%= link_to "Cards", cards_path, class:"dropdown-item" %></li>
+      <li>
+        <%= link_to orders_path, class:"dropdown-item" do %>
+          <h6><i class="bi bi-receipt"></i> Orders</h6>
```

Comment:
If we have pagination here, why we don't give page params here like in products ? `products_path(:per_page => 10)`